Makefile

```diff
@@ -448,28 +455,23 @@ staging: doc/CHANGES
 	# Delete the tmp files
 	cd ${TMPDIR} && /bin/rm -rf daikon daikon.dist old_tar.txt new_tar.txt
 
-# Copy the files in the staging area to the website.  This will copy
-# all of the files in staging, but will not delete any files in the website
-# that are not in staging.
+# Convert the staging area to the live release.
+# We do this by deleting the previous release directory and
+# renaming the staging directory to be the release directory.
 staging-to-www: $(STAGING_DIR)
-#copy the files
+	-chmod u+w $(WWW_PARENT)
 	-chmod -R u+w $(WWW_DIR)
-# # remove previous release archive files
-# 	\rm -f /cse/web/research/plse/daikon/download/daikon-*
-# # don't trash existing history directory
-# #	(cd $(STAGING_DIR) && tar cf - --exclude=history .) | (cd $(WWW_DIR) && tar xfBp -)
-# Remove previous files, except history and staging-daikon
-	(cd $(STAGING_DIR) && \ls | grep -v history | grep -v staging-daikon | xargs \rm -rf
-# Copy directory
-	rsync -Cavz --quiet $(STAGING_DIR) $(WWW_DIR)
-#	-chmod -R u-w $(WWW_DIR)
-# This command updates the dates and sizes in the various index files
-	html-update-link-dates $(DIST_DIR)/index.html
-# with new version number system, this is now done manually
-#	$(MAKE) update-dist-version-file
+	-chmod -R u+w $(STAGING_DIR)
+	# Remove previous release
```

Comment:
I would drop the comments here given that you have described the process in the comments at lines 459-460.